src/ray/protobuf/node_manager.proto

```diff
@@ -335,6 +335,14 @@ message NotifyGCSRestartRequest {}
 
 message NotifyGCSRestartReply {}
 
+message GetTaskResultRequest {
```

Comment:
I think this proto is a bit misleading since it sounds like we're getting the return value of the task. Can we rename it to `GetTaskFailureCause` or something similar? Or maybe even `GetWorkerFailureCause` since that's what we're actually interested in (it could've been an actor that failed too).